modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2976,6 +2976,25 @@ check-cron-expression=Check Cron Expression
+check-if-a-field-contains-a-specific-value-and-return-a-boolean=Check if a field a specific value and return a Boolean. If the field does contain the value, it is invalid.
+check-if-a-field-contains-a-specific-value-and-return-a-boolean-if-the-field-does-not-contain-the-value-it-is-invalid=Check if a field contains a specific value and return a Boolean. If the field does not contain the value, it is invalid.
+check-if-a-field-has-the-same-date-of-the-value=Check if a field has the same date of the value. If the field has the same value, it is valid.
+check-if-a-field-is-equal-to-a-specific-value-and-return-a-boolean=Check if a field is equal to a specific value and return a Boolean. If they are not equal, it is invalid.
+check-if-a-field-is-not-equal-to-a-specific-value-and-return-a-boolean=Check if a field is different from a specific value and return a Boolean. If it is, the field is invalid.
+check-if-a-number-field-is-a-decimal-and-return-a-boolean=Check if a number field is a decimal and return a Boolean. If it is, the field is invalid.
+check-if-a-number-field-is-a-integer-and-return-a-boolean=Check if a number field is an integer and return a Boolean. If it is, the field is invalid.
+check-if-a-number-field-greater-than-a-specific-numeric-value-and-return-a-boolean=Check if a number field is greater than a specific numeric value and return a Boolean. If it is, the field is invalid.
+check-if-a-number-field-greater-than-or-equal-to-a-specific-numeric-value-and-return-a-boolean=Check if a number field is greater than or equal to a specific numeric value and return a Boolean. If it is, the field is invalid.
+check-if-a-number-field-less-than-a-specific-numeric-value-and-return-a-boolean=Check if a number field is less than a specific numeric value and return a Boolean. If it is, the field is invalid.
+check-if-a-number-field-less-than-or-equal-to-a-specific-numeric-value-and-return-a-boolean=Check if a number field is less than or equal to a specific numeric value and return a Boolean. If it is, the field is invalid.
+check-if-a-text-field-is-an-email-and-return-a-boolean=Check if a text field is an email and return a Boolean. If the field does not match a specific email or email pattern, it is invalid. Valid entries must use standard email elements (e.g., @gmail, .com)
+check-if-a-text-field-is-a-URL-and-return-a-boolean=Check if a text field is a URL and return a Boolean. If the field does not match a URL pattern, it is invalid. Valid entries must use standard URL elements (e.g., http, ://, .com)
+check-if-a-text-field-is-empty-and-return-a-boolean=Check if a text field is empty and return a Boolean. Empty fields are invalid.
+check-if-a-text-field-is-different-to-a-specific-string-value-and-return-a-boolean=Check if a text field is different to a specific string value and return a Boolean. If they are not equal, it is invalid.
```

Comment:
@MariMalta It should be "is-different-from-a-specific...". The second part should be "If they are equal, it returns false"